source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/point/EntityType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Crown copyright
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.shared.domain.v2.point;
+
+/**
+ * Entity types of entity references
+ */
+public enum EntityType {
+  HABITAT,
```

Comment:
Would prefer `HABITAT_TYPE`: just habitat could also be a habitat area, and that's not what we're referencing).